core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -2686,7 +2690,7 @@ public <T extends RESTResponse> T execute(
         .execute(
             reqMatcher(
                 HTTPMethod.GET,
-                "v1/namespaces/newdb/tables/table",
+                "v1/namespaces/" + TABLE.namespace() + "/tables/" + TABLE.name(),
```

Comment:
can be replaced with `ResourcePaths.forCatalogProperties(Map.of()).table(TABLE)`